qiskit_optimization/algorithms/cplex_optimizer.py

```diff
@@ -116,33 +139,25 @@ def solve(self, problem: QuadraticProgram) -> OptimizationResult:
             QiskitOptimizationError: If the problem is incompatible with the optimizer.
         """
 
-        # convert to CPLEX problem
-        cplex = problem.to_docplex().get_cplex()
-
-        # set display setting
-        if not self.disp:
-            cplex.set_log_stream(None)
-            cplex.set_error_stream(None)
-            cplex.set_warning_stream(None)
-            cplex.set_results_stream(None)
-
-        # solve problem
-        try:
-            cplex.solve()
-        except CplexSolverError as ex:
-            raise QiskitOptimizationError(str(ex)) from ex
-
-        # process results
-        sol = cplex.solution
-
-        # create results
-        result = OptimizationResult(
-            x=sol.get_values(),
-            fval=sol.get_objective_value(),
-            variables=problem.variables,
-            status=self._get_feasibility_status(problem, sol.get_values()),
-            raw_results=sol,
-        )
-
-        # return solution
+        mod = problem.to_docplex()
+        sol = mod.solve(log_output=self._disp, cplex_parameters=self._cplex_parameters)
```

Comment:
Don't really get, why is this better rather than calling cplex directly? Just curious...

Reply:
I think there are two reasons
1. docplex is more pythonic than `cplex` and easier to maintain.
2. we need to refer only docplex manual to maintain this code. Previously we need to refer both manuals of cplex and docplex.
Basically, I prefer to hide `cplex`

Reply:
OK, thanks